Test/RivetTest/Functions/Start-RivetTest.ps1

```diff
@@ -13,7 +13,9 @@ function Start-RivetTest
 
         [String[]] $ConfigurationDatabase,
 
-        [int] $CommandTimeout = 30
+        [int] $CommandTimeout = 30,
+
+        [Switch] $Pester5

```

Comment:
Omit this switch. Instead, you can check if you're running Pester 5 by looking for the `FullName` property on `$TestDrive`:

```
if ($testDirectory | Get-Member -Name 'FullName')
{
    $testDirectory = $testDirectory.FullName
}
```